napari/layers/_image_layer/model.py

```diff
@@ -351,35 +317,42 @@ def interpolation_functions(self):
     def _clim_range_default(self):
         return [float(self.image.min()), float(self.image.max())]
 
-    def get_value(self, position, indices):
+    def get_value(self):
         """Returns coordinates, values, and a string for a given mouse position
         and set of indices.
 
-        Parameters
-        ----------
-        position : sequence of two int
-            Position of mouse cursor in canvas.
-        indices : sequence of int or slice
-            Indices that make up the slice.
-
         Returns
         ----------
         coord : sequence of int
             Position of mouse cursor in data.
         value : int or float or sequence of int or float
             Value of the data at the coord.
-        msg : string
-            String containing a message that can be used as
-            a status update.
         """
-        transform = self._node.canvas.scene.node_transform(self._node)
-        pos = transform.map(position)
-        pos = [np.clip(pos[1], 0, self._image_view.shape[0]-1),
-               np.clip(pos[0], 0, self._image_view.shape[1]-1)]
-        coord = list(indices)
-        coord[-2] = int(pos[0])
-        coord[-1] = int(pos[1])
+        coord = np.round(self.coordinates).astype(int)
+        coord[-2] = np.clip(coord[-2], 0, self._image_view.shape[0]-1)
+        coord[-1] = np.clip(coord[-1], 0, self._image_view.shape[1]-1)
+
```

Comment:
```
coord = np.round(self.coordinates).astype(int)
coord[-2:] = np.clip(coord[-2:], 0, np.asarray(self.image_view.shape) - 1)
```

As mentioned elsewhere, it'd be good here to mention the coordinate space when you use it.

Reply:
i've improved the doc string

Reply:
and made the change to clip